10-inference.Rmd

```diff
@@ -83,124 +87,120 @@ The population consists of all studio apartment rentals in Vancouver, and the po
 
 ### Sampling distributions for proportions
+We will look at an example using data from [Inside Airbnb](http://insideAirbnb.com/). Airbnb is an online marketplace for arranging places to stay. The data set, `airbnb`, contains listings for Vancouver, Canada, in September 2020. Our data includes an ID number, neighbourhood, type of room, the number of people the rental accommodates, number of bathrooms, bedrooms, beds, and the price per night. 
 
-Let's start with an illustrative (and tasty!) example. Timbits are 
-bite-sized doughnuts sold at Tim Hortons, a popular Canadian-based fast-food restaurant
-chain founded in Hamilton, Ontario, Canada. 
-
-```{r 11-timbits-picture, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "A box of Timbits", fig.retina = 2, out.width = "30%"}
-knitr::include_graphics("img/timbits.jpg")
+```{r 11-example-means1, echo = FALSE, message = FALSE, warning = FALSE}
+airbnb <- read_csv("data/listings.csv") %>%
+  select(id, neighbourhood = neighbourhood_cleansed, room_type, accommodates, bathrooms = bathrooms_text, bedrooms, beds, price) %>%
+  mutate(price = as.numeric(str_remove(price, "[$]"))) %>%
+  na.omit()
+airbnb <- airbnb %>%
+  mutate(id = 1:nrow(airbnb))
 ```
 
+```{r 11-vancouver-city, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Vancouver, BC Canada", fig.retina = 2, out.width="880"}
+knitr::include_graphics("https://upload.wikimedia.org/wikipedia/commons/7/76/Vancouver_City_2_by_ajithrajeswari.jpg")
+```
 
-Suppose we wanted to estimate the true proportion of chocolate doughnuts at Tim
-Hortons restaurants. Now, of course, we (the authors!) do not have access to the true population.
-So for this chapter, we created a fictitious box of 10,000 Timbits with two flavours&mdash;old-fashioned 
-and chocolate&mdash;as our population, and use this to illustrate
-inferential concepts. Below we have a `tibble()` called `virtual_box` with a Timbit ID and flavour as our columns. We have also loaded our necessary packages: `tidyverse` and the `infer` package, which we will need to perform sampling later in the chapter.  
-
-```{r 11-example-proportions, echo = FALSE, message = FALSE, warning = FALSE}
-library(gridExtra)
-set.seed(1234)
-virtual_box <- tibble(
-  timbit_id = seq(1, 10000, by = 1),
-  flavour = factor(rbinom(10000, 1, 0.63),
-    labels = c("old fashioned", "chocolate")
-  )
-)
+```{r}
+airbnb
 ```
+Suppose the city of Vancouver wants to know how many Airbnb places are listed as entire homes and apartments (rather than as private or shared rooms). Therefore they may want to estimate the true proportion of all Airbnb listings where the "type of place" is listed as "entire home or apartment." Of course, we usually do not have access to the true population, but here let's imagine (for learning purposes) that our data set represents the population of all Airbnb rental listings in Vancouver, Canada. We can find the proportion of listings where `room_type == "Entire home/apt"`.
```

Comment:
why would vancouver want to know this? would be good to have some motivation

Reply:
I added something about using the information to city bylaws -- I don't know if that is realistic or not... Feel free to let me know if I should change that!